src/main/resources/org/jenkinsci/plugins/github_branch_source/GitHubAppCredentials/config.jelly

```diff
@@ -22,12 +22,21 @@
   </f:entry>
 
   <f:advanced>
```

Comment:
I'm not sure if we should still hide these options behind the advanced dropdown or not. Any thoughts?